_component_design/table-control.md

```diff
@@ -0,0 +1,427 @@
+- **Search (Optional)**: The top of the table can include a search bar that a user can use to locate records.
+- **Show/Hide (Required/Optional)**: The user can select to show or hide columns from view. Show/hide enables the user to select which data they want to view.
+- **Range indicator (Required/Optional)**: The range indicator displays which records are displayed currently in the table. For example, **Showing 20-30 of 450** means that out of 450 objects, the only ones being displayed are records 20 through 30.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Filter buttons trancated/truncated-compact variations"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/table-controls/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use these specifications when you construct table top controls.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Sorting states and icons"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/table-controls/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Spacing
+
+Refer to these spacing guidelines when you construct table top controls.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Sorting states and icons"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/table-controls/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Table header control specifications
+
+Use the following specifications when you construct table header controls.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+The header of the table can include the following controls:
+
+- **Bulk selection (Optional)**: When the user clicks the bulk selection checkbox, all records in on the current page are selected.
+- **Sorting (Optional)**: Users can sort the table rows to appear either in ascending order or descending order.
+
```

Comment:
default sort, make sure the default sort is based on the column that makes the most sense. For example, if tickets, then default sort by date from least to most recent. Should be logical for the information needs of the user. Arrow to change based on the direction of the sort, either ascending or descending.